src/objects/button-group/button-group.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import demo from './demo/demo.twig';
+
+<Meta
+  title="Objects/Button Group"
+  argTypes={{
+    tight: {
+      type: { name: 'boolean' },
+      defaultValue: false,
+      description: 'Controls the tight-spacing modifier',
+    },
+    grow: {
+      type: { name: 'boolean' },
+      defaultValue: false,
+      description: 'Controls the button full width modifier',
+    },
+    count: {
+      type: { name: 'number' },
+      defaultValue: 2,
+      description: 'Number of buttons to render',
+      control: {
+        type: 'range',
+        min: 1,
+        max: 10,
+        step: 1,
+      },
+    },
+  }}
+/>
+
+# Button Group
+
+The Button Group layout object can be used for aligning a group of related buttons.
+
+## Default
+
+```php
```

Comment:
I'm about to submit a PR that adds support for `twig` highlights to code blocks. If you update your code blocks to `twig` instead of `php`, it'll save us some work down the road.

Reply:
🔥 https://github.com/cloudfour/cloudfour.com-patterns/pull/1240/commits/b117a76bbabd56c8853e5fff71eb14eb0e033b80